bin/emqx

```diff
@@ -421,16 +425,14 @@ generate_config() {
     call_hocon -v -t "$NOW_TIME" -s "$SCHEMA_MOD" -c "$EMQX_ETC_DIR"/emqx.conf -d "$DATA_DIR"/configs generate
 
     ## filenames are per-hocon convention
-    local CONF_FILE="$CONFIGS_DIR/app.$NOW_TIME.config"
-    local HOCON_GEN_ARG_FILE="$CONFIGS_DIR/vm.$NOW_TIME.args"
+    CONF_FILE="$CONFIGS_DIR/app.$NOW_TIME.config"
+    ARGS_FILE="$CONFIGS_DIR/vm.$NOW_TIME.args"
 
     # This is needed by the Elixir scripts.
     # Do NOT append `.config`.
     RELEASE_SYS_CONFIG="$CONFIGS_DIR/app.$NOW_TIME"
     export RELEASE_SYS_CONFIG
```

Comment:
I just noticed that this was used when we used the Elixir generated `bin/emqx`, but this is no longer the case.  We can remove those lines now.  :smile_cat: